test/liquid_voting/voting_results_test.exs

```diff
@@ -26,19 +26,30 @@ defmodule LiquidVoting.VotingResultsTest do
       assert against == 0
     end
 
-    test "returns the same result struct for a given proposal_url", context do
-      %Result{id: id} =
-        VotingResults.calculate_result!(context[:proposal_url], context[:organization_uuid])
+    test "inserts data into existing result, for a given proposal_url && organization", context do
+      VotingResults.calculate_result!(context[:proposal_url], context[:organization_uuid])
+
+      results1 = VotingResults.list_results(context[:organization_uuid])
+
+      %Result{uuid: uuid, in_favor: in_favor} = Enum.at(results1, 0)
 
       insert(:vote,
         proposal_url: context[:proposal_url],
         organization_uuid: context[:organization_uuid]
       )
 
-      %Result{id: new_id} =
-        VotingResults.calculate_result!(context[:proposal_url], context[:organization_uuid])
+      VotingResults.calculate_result!(context[:proposal_url], context[:organization_uuid])
+
+      results2 = VotingResults.list_results(context[:organization_uuid])
+
+      %Result{uuid: new_uuid, in_favor: new_in_favor} = Enum.at(results2, 0)
 
-      assert id == new_id
+      # Only one result has been added to the list of results
+      assert Enum.count(results1) == Enum.count(results2)
```

Comment:
I think asserting the uuid of the returned result is still the same is enough, no need to list results.

Reply:
this is the (other of the 2) upsert issues, previously described. The problem is exactly that: the uuids won't match using the test as previously was.